.github/workflows/integration-test.yml

```diff
@@ -5,12 +5,9 @@ name: Run Integration Tests
 
 on:
   push:
-    branches:
-      - dev
-    paths-ignore:
-      - 'README.md'
-      - '.github/**'
-      - 'local-dev/**'
```

Comment:
Why remove the paths-ignore section?

Reply:
local-dev has the postgres init script. We actually care about if that changes, as that's actually used in this workflow for setting up the db.
The workflow itself is under .github. If we change the workflow, the test should run.
README.md doesn't matter, but neither does DEVELOPMENT.md. I'm not sure it's worth trying to keep an exhaustive list of not-actually-important files to save a few github action runs. So, meh.